pylint/checkers/python3.py

```diff
@@ -598,6 +598,12 @@ class Python3Checker(checkers.BaseChecker):
             "variables will be deleted outside of the "
             "comprehension.",
         ),
+        "C1601": (
```

Comment:
The checks in this file are disabled by default since they are meant for Python 3 porting, and this one does not have to do with Python 3 porting at all.

Reply:
I merged too fast, do you want me to revert ?

Reply:
No it's fine, just submitted a PR to move it.